File: web3rpc/rpc-specs/code-samples/python/personal/openWallet.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

url = "keystore://"
passphrase = "helloWorld"

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.open_wallet(url, passphrase)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/openWallet.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const url = "keystore://";
  const passphrase = "gr8=B!0@uc$b";

  provider.personal
    .openWalconst(url, passphrase, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
